src/sentry/models/groupassignee.py

```diff
@@ -34,6 +35,37 @@
 
 
 class GroupAssigneeManager(BaseManager):
+    def get_assigned_to_data(
+        self, assigned_to: Team | RpcUser, assignee_type: str, extra: Dict[str, str] | None = None
+    ) -> Dict[str, str | Any | None]:
```

Comment:
`Any` encapsulates `str` and `None`, so should this just be `Dict[str, Any]`? https://docs.python.org/3/library/typing.html#the-any-type